index.html

```diff
@@ -1,116 +1,229 @@
+          <a href=""><i class="bi bi-twitter"></i></a>
+        </li>
+        <li class="p-1"><a href="">English</a></li>
+        <li class="p-1"><a href="">My Page</a></li>
+        <li class="p-1"><a href="">logout </a></li>
       </ul>
-    </div>
-  </nav>
 
-  <!-- Home Page-->
-  <div class="home">
-    <h1>The Grape Escape: Ultimate Science of Wine 2023!</h1>
-    <p>We invite you to the upclose Chemistry of wine making and tasting.</p>
-    <p>This, you do not want to miss. <br><span>2023.10.15(THU) ~ 16(FRI)</span>
-      <br><span>@ Stellenbosch Wine Farm, Cape Town</span> </p>
-  </div>
+      <!-- navigation -->
 
-  <!-- Program Page -->
-  <div class="program">
-    <h2>Main Program</h2>
-    <div class="program-tiles">
-      <div class="program-tile">
-        <img src="./assets/program_icon_01.png" alt="Program 1">
-        <h3>Lecture</h3>
-        <p>Listen to the speakers from various Universities about wine.</p>
-      </div>
-      <div class="program-tile">
-        <img src="./assets/program_icon_05.png" alt="Program 2">
-        <h3>CC Exhibition</h3>
-        <p>Appreciate various creations sharing wine brewing skills,</p>
-      </div>
-      <div class="program-tile">
-        <img src="./assets/program_icon_04.png" alt="Program 3">
-        <h3>Forum</h3>
-        <p>Have the time to share your thoughts with experts for each topic. 3</p>
-      </div>
-      <div class="program-tile">
-        <img src="./assets/program_icon_01.png" alt="Program 4">
-        <h3>Workshop</h3>
-        <p>Try brewing your own wine using traditional methods.</p>
-      </div>
-      <div class="program-tile">
-        <img src="./assets/program_icon_04.png" alt="Program 5">
-        <h3>CC Ignite</h3>
-        <p>Get opportunitiy to sniff and sip wine on a blindfold for a price.</p>
-      </div>
-    </div>
-  </div>
+      <nav class="px-[10%] md:flex h-[55px] justify-between shadow-xl">
+        <i class="menulist bi bi-list md:hidden p-4 text-[40px] pl-0"></i>
+        <a class="p-3 hidden md:flex" href="index.html"
+          ><img class="logo" src="assets/wine-glass.png" alt="" />wine~up</a
+        >
+        <ul class="items md:flex items-center justify-center">
+          <li class="close"><i class="bi bi-x-lg"></i></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="about.html">About</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Program</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">Join</a></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Sponsor</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">News</a></li>
+          <li
+            class="item p-1 m-1 hover:text-[#ec5242] border-solid border-4 border-[#ec5242]"
+          >
+            <a href="">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
The mobile navbar is not working. Kindly fix this issue. I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where you can verify the problem.